src/Umbraco.Web.UI.Client/src/views/packages/edit.html

```diff
@@ -19,10 +19,10 @@
 
                 <div class="umb-expansion-panel">
 
-                    <div class="umb-expansion-panel__header" ng-click="vm.propertiesOpen = !vm.propertiesOpen">
-                        <div><localize key="packager_packageProperties">Package Properties</localize></div>
+                    <button type="button" aria-expanded="{{vm.propertiesOpen}}" class="btn-reset umb-expansion-panel__header" ng-click="vm.propertiesOpen = !vm.propertiesOpen">
+                        <span><localize key="packager_packageProperties">Package Properties</localize></span>
                         <umb-icon icon="{{vm.propertiesOpen ? 'icon-navigation-up' : 'icon-navigation-down'}}" class="umb-expansion-panel__expand" ng-class="{'icon-navigation-down': !vm.propertiesOpen, 'icon-navigation-up': vm.propertiesOpen}">&nbsp;</umb-icon>
```

Comment:
Hey @BatJan, same with these icons - `ng-class` attribute is not required. Tried to push and fix it for you, but got the same git issue